docs/sources/shared/index.md

```diff
@@ -0,0 +1,3 @@
+---
+headless: true
+---
```

Comment:
@knylander-grafana Yeah this is a painful one. Prettier wants this to end in a newline but the bot that reports the suggestion is stripping it. 

I've opened https://github.com/grafana/writers-toolkit/issues/307 to track this because it's come up a couple times.

```suggestion
---

```